### PR TITLE
Add a column for number of failed tests(#170)

### PR DESCRIPTION
**Pull request by itzikb**
A patch to add the number of failed tempest test for the last build of each job

### CODE REVIEW

**Review by agarcia64 — APPROVED** [review]
Tested on my machine, everything is working as expected. In addition, the code looks clean and consistent. Looks good to me!

File: report.py [hunk 1]
```diff
@@ -138,15 +138,15 @@ def run_report(config, blockers, server, header, test_email, no_email, template_
...
+				'other': other,
+				'tempest_tests_failed': jenkins_api_info['tempest_tests_failed']
 			}
 
 			# append row to rows
 			rows.append(row)
 
 	# sort rows by descending OSP version
 	rows = sorted(rows, key=lambda row: row['osp_version'], reverse=True)
-
```
Comment (amodi5) [inline report.py:149]: nit: add the blank line

**Review by amodi5 — COMMENTED** [review]
LGTM!

**Review by nathan-weinberg — CHANGES_REQUESTED** [review]
Two small requests:
1. Could we implement this for reminders as well?
2. If there value is not "N/A", could we link to the tempest results page on Jenkins? (if this is non-trivial, it does not need to block merging, but please open an RFE to implement this feature later)

File: report.py (hunk 1 above)
Comment (itzikb) [reply report.py:149]: Done

**Comment by itzikb** [comment]
Done
> Two small requests:
> 
> 1. Could we implement this for reminders as well?
> 2. If there value is not "N/A", could we link to the tempest results page on Jenkins? (if this is non-trivial, it does not need to block merging, but please open an RFE to implement this feature later)

**Review by nathan-weinberg — APPROVED** [review]
LGTM! Will leave up for now so others can review but will merge by EOD :+1:

**Pull request merged**